packages/sitecore-jss-nextjs/src/index.ts

```diff
@@ -1,3 +1,4 @@
+/** ClientError is @deprecated. import 'GraphQLClientError' from '@sitecore-jss/sitecore-jss-nextjs/graphql' instead */
```

Comment:
Need to apply this deprecated comment _only_ to the `ClientError`?

Reply:
Yeah I moved it to the top and made it clear that only `ClientError` is deprecated.
But i think moving it just above `ClientError` would make better sense.